
Qt-3.1.2 does not has this swapped cursors ...

  - Rene Rebe <rene@rocklinux.org>

--- koffice-1.2.94/kword/kwdoc.cc	2003-10-27 14:21:55.000000000 +0100
+++ koffice-1.2.94-fixed/kword/kwdoc.cc	2003-12-10 08:05:39.000000000 +0100
@@ -3278,13 +3278,13 @@
         return Qt::sizeVerCursor;
     case MEANING_RESIZE_COLUMN:
         // Bug in (at least) Qt-3.1.1 : Qt::splitVCursor and Qt::splitHCursor are swapped!
-#if QT_VERSION < 0x030200
+#if QT_VERSION < 0x030101
         return Qt::splitVCursor;
 #else
         return Qt::splitHCursor;
 #endif
     case MEANING_RESIZE_ROW:
-#if QT_VERSION < 0x030200
+#if QT_VERSION < 0x030101
         return Qt::splitHCursor;
 #else
         return Qt::splitVCursor;
